core/test/results/sample_v2.json

```diff
@@ -7697,12 +7697,6 @@
         "duration": 100,
         "entryType": "measure"
       },
-      {
-        "startTime": 0,
-        "name": "lh:computed:EntityClassification",
-        "duration": 100,
-        "entryType": "measure"
-      },
```

Comment:
What's the deal w/ this? Were we computing `EntityClassification` multiple times?

Reply:
yeah, see how `data` was being passed directly in the line i changed

Reply:
We should fill out the dependencies in `EntityClassification` to prevent this in the future. i.e. replace `null` here:

https://github.com/GoogleChrome/lighthouse/blob/1f6fab00d45cd6e6aa6e4eeef6b216a76dd1e7cf/core/computed/entity-classification.js#L101

Reply:
cc @alexnj

Reply:
https://github.com/GoogleChrome/lighthouse/pull/14732